app/config.go

```diff
@@ -0,0 +1,75 @@
+package gaia
+
+import (
+	"flag"
+
+	"github.com/cosmos/cosmos-sdk/types/simulation"
+)
+
+// List of available flags for the simulator
+var (
+	FlagGenesisFileValue        string
+	FlagParamsFileValue         string
+	FlagExportParamsPathValue   string
+	FlagExportParamsHeightValue int
+	FlagExportStatePathValue    string
+	FlagExportStatsPathValue    string
+	FlagSeedValue               int64
+	FlagInitialBlockHeightValue int
+	FlagNumBlocksValue          int
+	FlagBlockSizeValue          int
+	FlagLeanValue               bool
+	FlagCommitValue             bool
+	FlagOnOperationValue        bool // TODO: Remove in favor of binary search for invariant violation
+	FlagAllInvariantsValue      bool
+
+	FlagEnabledValue     bool
+	FlagVerboseValue     bool
+	FlagPeriodValue      uint
+	FlagGenesisTimeValue int64
+)
+
+// GetSimulatorFlags gets the values of all the available simulation flags
+func GetSimulatorFlags() {
```

Comment:
no need for all these flags as they are imported from the SDK simapp package

Reply:
Thanks! Good catch!!